docs/rpc_api_versioning.md

```diff
@@ -269,6 +270,88 @@ the JSON-RPC API documentation will be updated to include documentation for
     // ...
 ```
 
+<a name="Websockets" />
+
+### 8. Websocket Endpoints & Example
+Websocket endpoints will be versioned using the accessibilty scheme outlined 
+above. Using `rescan` websocket endpoint and the introduction of `rescanv2` 
+as an example:
+
+```go
+// Rescan websocket handler registrations.
+var wsHandlersBeforeInit = map[types.Method]wsCommandHandler{	
+	// ...
+	"rescan":                      handleRescanV2,
+	"rescanv1":                    handleRescan,
+	"rescanv2":                    handleRescanV2,
+	// ...
+}
+```
+
+```go
+// Versioned JSON-RPC endpoint command definition.
+
+// RescanCmd defines the rescan JSON-RPC command.
+type RescanCmd struct {
+	BlockHashes []string
+}
+
+// RescanV2Cmd defines the rescanv2 JSON-RPC command.
+type RescanV2Cmd struct {
+	From string
+	To   string
+}
+```
+
+```go
+// Rescan JSON-RPC result type registrations.
+var rpcResultTypes = map[types.Method][]interface{}{
+    // Websocket commands.
+
+    // ...
+	"rescan":                        nil,
+    "rescanv1":                      nil,
+    "rescanv2":                      nil,
+    // ...
+}
+```
+
+```go
+// Rescan Websocket type registrations.
+func init() {
+	// The commands in this file are only usable by websockets.
+	flags := dcrjson.UFWebsocketOnly
+
+    //...
+    dcrjson.MustRegister(Method("rescan"), (*RescanV2Cmd)(nil), flags)
+    dcrjson.MustRegister(Method("rescanv1"), (*RescanV1Cmd)(nil), flags)
+    dcrjson.MustRegister(Method("rescanv2"), (*RescanV2Cmd)(nil), flags)
+    //...
+}
+```
+
+```go
+// handleRescan implements the rescan command extension for websocket
+// connections.
+func handleRescan(wsc *wsClient, icmd interface{}) (interface{}, error) {
+	cmd, ok := icmd.(*types.RescanCmd)
+	if !ok {
+		return nil, dcrjson.ErrRPCInternal
+	}
+    // ...
+}
+
+// handleRescanV2 implements the rescanv2 command extension for websocket
+// connections.
+func handleRescanV2(wsc *wsClient, icmd interface{}) (interface{}, error) {
+	cmd, ok := icmd.(*types.RescanV2Cmd)
+	if !ok {
+		return nil, dcrjson.ErrRPCInternal
+	}
+    // ...
+}
+```
+
 With the outlined schemes adhered to, the versioned endpoint should be ready 
```

Comment:
This last sentence is clunky, and I'm not sure what you're trying to say. Is this a list of remaining steps to take before the new versioned endpoints API can be active? I don't really understand the second half of the sentence, but I gave it my best shot. Please feel free to modify:

Once all `dcrd` JSON-RPC API methods are updated using the schema outlined above, the versioned endpoints will be ready for use. However, `dcrctl` and other consumers will need to update their `rpc/jsonrpc/types` dependencies and logic.

Reply:
I'm saying dcrctl needs to be rebuilt with the updated jsonrpc types to be able to access the new versioned commands.